k8s/base/app/kustomization.yaml

```diff
@@ -4,4 +4,4 @@ resources:
   - service.yaml
   - redis-service.yaml
   - qcluster-deployment.yaml
-
+  - storage-invoice-cronjob.yaml
```

Comment:
the file you created is called `invoice-cronjob.yaml`. Make sure this builds when you run `kustomize build`.